api/android/api/src/main/jni/nnstreamer-native-api.c

```diff
@@ -572,11 +576,15 @@ nnstreamer_native_initialize (void)
   /* register nnstreamer plugins */
   GST_PLUGIN_STATIC_REGISTER (nnstreamer);
 
-  /* tensor-filter sub-plugin for tensorflow-lite */
-  init_filter_tflite ();
+  /* Android MediaCodec */
+  GST_PLUGIN_STATIC_REGISTER (amcsrc);
 
+  /* tensor-filter sub-plugins */
+  init_filter_cpp ();
+  init_filter_custom ();
+  init_filter_custom_easy ();
+  init_filter_tflite ();
```

Comment:
We need some mechanism added (preferably with some internal APIs) to prevent calling init/fini twice in a given process.

Reply:
add flag and global lock to check the plugin registration.